build.fsx

```diff
@@ -119,6 +122,11 @@ Target "Build" (fun _ ->
     |> ignore
 )
 
+Target "Lint" (fun _ ->
+    !! "**/*.fsproj" 
+    |> Seq.iter (FSharpLint (fun options -> { options with FailBuildIfAnyWarrnings = true })) 
```

Comment:
@duckmatt I just noticed a typo. Parameter should be called `FailBuildIfAnyWarnings`